getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Get started with VPP networking
+description: Install Calico with the VPP dataplane on a Kubernetes cluster.
+canonical_url: '/getting-started/kubernetes/vpp/getting-started'
+---
+
+
+### Big picture
+
+Install {{site.prodname}} and enable the tech preview of the VPP dataplane.
+
+> **Warning!** The VPP dataplane is a tech preview and should not be used in production clusters. It has had limited testing and it will contain bugs (please report these on the [Calico Users slack](https://calicousers.slack.com/archives/C017220EXU1) or [Github](https://github.com/projectcalico/vpp-dataplane/issues)).  In addition, it does not support all the features of {{site.prodname}} and it is currently missing some security features such as Host Endpoint policies.
+{: .alert .alert-danger }
+
+### Value
+
+The VPP dataplane mode has several advantages over standard Linux networking pipeline mode:
+
+* Scales to higher throughput, especially with WireGuard encryption enabled
+* Further improves encryption performance with IPsec
+* Native support for Kubernetes services without needing kube-proxy, which:
+  * Reduces first-packet latency for packets to services
+  * Preserves external client source IP addresses all the way to the pod
+
+The VPP dataplane is entirely compatible with the other {{site.prodname}} dataplanes, meaning you can have a cluster with VPP-enabled nodes along with regular nodes. This makes it easy to migrate a cluster from Linux or eBPF networking to VPP networking.
+
+In the future, the VPP dataplane will offer additional features for network-intensive applications, such as providing `memif` userspace packet interfaces to the pods (instead of regular Linux network devices).
+
+Trying out the tech preview will give you a taste of these benefits and an opportunity to give feedback to the VPP dataplane team.
+
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **calico/node**
+- **VPP dataplane**
+
+### Concepts
+
+#### VPP
+
+The Vector Packet Processor (VPP) is a high-performance, open-source userspace network dataplane written in C, developed under the [fd.io](https://fd.io) umbrella. It supports many standard networking features (L2, L3 routing, NAT, encapsulations), and is easily extensible using plugins. The VPP dataplane uses plugins to efficiently implement Kubernetss services load balancing and {{site.prodname}} policies.
+
+### How to
+
+This guide details two ways to install Calico with the VPP dataplane:
+- On a [managed EKS cluster](#install-calico-with-the-vpp-dataplane-on-an-eks-cluster). This is the option that requires the least configuration.
+- On [any Kubernetes cluster](#install-calico-with-the-vpp-dataplane-on-any-kubernetes-cluster).
+
+In both cases, here are the details of what you will get:
+
+{% include geek-details.html details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:IPIP,Routing:BGP,Datastore:Kubernetes' %}
+
```

Comment:
Recommend using our inline tabs...both instructions are long so I almost started the next section until I realized I had finished. 

{% tabs %}
  <label:name,active:true>
<%

%>
  '<label:name2>'

<%

%>
{% endtabs %}

Reply:
Switched to tabs :)